l5kit/l5kit/data/filter.py

```diff
@@ -104,3 +104,54 @@ def filter_agents_by_frames(frames: np.ndarray, agents: np.ndarray) -> List[np.n
     if frames.shape == ():
         frames = frames[None]  # add and axis if a single frame is passed
     return [agents[slice(*frame["agent_index_interval"])] for frame in frames]
+
+
+def filter_tl_faces_by_frames(frames: np.ndarray, tl_faces: np.ndarray) -> List[np.ndarray]:
+    """
+    Get a list of traffic faces array, one array per frame.
+    This functions mimics `filter_agents_by_frames` for traffic light faces
+
+    Args:
+        frames (np.ndarray): an array of frames
+        tl_faces (np.ndarray): an array of traffic light faces
+
+    Returns:
+        List[np.ndarray] with the traffic light faces divided by frame
+    """
+    return [tl_faces[slice(*frame["tl_faces_index_interval"])] for frame in frames]
+
+
+def get_tl_faces_active(tl_faces: np.ndarray) -> np.ndarray:
```

Comment:
Wouldn't it be nicer to have a single function get_tl_faces_by_status and then pass the status as parameter?

Reply:
it's more error prone as you write the strings literal in the code. The correct solution is probably a `NamedTuple` or something like that (the same way you would handle colours). I'd leave it for another PR

Reply:
I'll go on and do it since there is quorum on this. We will refactor it later to support enum